crates/wasmtime/src/runtime/vm.rs

```diff
@@ -243,6 +243,28 @@ pub fn page_size() -> usize {
     };
 }
 
+/// Is `bytes` a multiple of the host page size?
+pub fn usize_is_multiple_of_host_page_size(bytes: usize) -> bool {
+    bytes % host_page_size() == 0
+}
+
+/// Round the given byte size up to a multiple of the host OS page size.
+pub fn round_u64_up_to_host_pages(bytes: u64) -> u64 {
+    let page_size = u64::try_from(crate::runtime::vm::host_page_size()).unwrap();
+    debug_assert!(page_size.is_power_of_two());
+    bytes
+        .checked_add(page_size - 1)
+        .map(|val| val & !(page_size - 1))
+        .unwrap_or(u64::MAX / page_size + 1)
```

Comment:
Should this be an `unwrap_or_else` to avoid always computing the default?
```suggestion
        .unwrap_or_else(|| u64::MAX / page_size + 1)
```
Also, could you add a comment that explains the default?

Reply:
I think it probably doesn't matter much either way in this case, since there isn't anything here that could prevent LLVM from cleaning this up itself (like a function call) but happy to make the change so that future readers don't ahve to reason through that.